x-pack/test/accessibility/config.ts

```diff
@@ -15,17 +15,18 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     ...functionalConfig.getAll(),
 
     testFiles: [
-      require.resolve('./apps/login_page'),
-      require.resolve('./apps/home'),
-      require.resolve('./apps/grok_debugger'),
-      require.resolve('./apps/search_profiler'),
-      require.resolve('./apps/uptime'),
-      require.resolve('./apps/spaces'),
-      require.resolve('./apps/advanced_settings'),
-      require.resolve('./apps/dashboard_edit_panel'),
-      require.resolve('./apps/users'),
-      require.resolve('./apps/roles'),
-      require.resolve('./apps/kibana_overview'),
+      // require.resolve('./apps/login_page'),
+      // require.resolve('./apps/home'),
+      // require.resolve('./apps/grok_debugger'),
+      // require.resolve('./apps/search_profiler'),
+      // require.resolve('./apps/uptime'),
+      // require.resolve('./apps/spaces'),
+      // require.resolve('./apps/advanced_settings'),
+      // require.resolve('./apps/dashboard_edit_panel'),
+      // require.resolve('./apps/users'),
+      // require.resolve('./apps/roles'),
+      // require.resolve('./apps/kibana_overview'),
+      require.resolve('./apps/ingest_node_pipelines'),
```

Comment:
Will want to uncomment this before merging

Reply:
yep same ^

Reply:
Fixed